src/Main.java

```diff
@@ -1,5 +1,17 @@
 public class Main {
     public static void main(String[] args) {
+        Author aa = new Author("Владимир", "Маяковский");
+        Book book = new Book("Сборник стихов", 2010, aa);
+        book.setYear(2011);
+        System.out.println("Книга называется - " + book.getTitle() + ", она издана в " + book.getYear() + " году. А автора этой книги зовут " + aa.getName() + " " + aa.getSurname());
```

Comment:
```suggestion
        System.out.println("Книга называется - " + book.getTitle() + ", она издана в " + book.getYear() + " году. А автора этой книги зовут " + book.getName().getName() + " " + book.getName().getSurname());
```
автора надо брать из книги, а не какого-то левого